tests/test_real_browsers.py

```diff
@@ -191,3 +192,17 @@ async def test_request(session):
     url = "/window/handles"
     handles = await session.request(url)
     assert len(handles) == 1
+
+
+@pytest.mark.parametrize(
+    "window_type", [None, WindowType.tab.value, WindowType.window.value]
+)
+async def test_new_window(session, window_type):
+    new_window_data: Dict[str, Any] = await session.new_window(window_type)
+    assert new_window_data["type"] == (
+        window_type if window_type is not None else WindowType.tab.value
```

Comment:
The spec says that `type=None` behaviour is up to browser implementation...
I suppose desktop browsers prefer tabs nowadays, but I wonder if that's the case for all supported browsers 🤔

Reply:
You are absolutely right. Moreover there are some failing checks about this. Because for IE driver "type" is a required field.